### PR TITLE
Fix axis 18 WIP

### PR DESCRIPTION
**Pull request by bazkiaei**
Learning how to work with pull request

### CODE REVIEW

**Comment by AnthonyHorton** [comment]
Should also improve the tests, to ensure errors of this type will get caught in future. Currently the tests of the `pixellated` method in `gunagala/tests/test_psf.py` all use square images so that the order doesn't matter. Should make at least some of those rectangular, and check that the returned image is the right shape.

Worse, there are currently no tests of the image simulation features in `gunagala/tests/test_imager.py` yet. That's beyond the scope of this PR though, we'll fix that afterwards.

File: gunagala/camera.py [hunk 1]
```diff
@@ -39,6 +39,7 @@ class Camera:
     Resolution : astropy.units.Quantity
         Two element Quantity containing the number of pixels across
         the image sensor in both horizontal & vertical directions.
```
Comment (AnthonyHorton) [inline gunagala/camera.py]: Replace "horizontal & vertical direction" with "vertical & horizontal directions" just to make this really clear.

File: gunagala/tests/test_psf.py [hunk 2]
```diff
@@ -88,15 +88,30 @@ def test_shape(psf):
 
 
 def test_pixellated(psf):
```
Comment (AnthonyHorton) [inline gunagala/tests/test_psf.py]: These tests all need unique names that describe the particular thing that they're testing, e.g. `test_pixellated_square`, `test_pixellated_rectangle`, `test_pixellated_invalid_shape`, etc.

**Comment by AnthonyHorton** [comment]
Don't leave blank lines between the `@pytest.mark.parameterize` decorator and the test function declaration that follows it. The decorator is effectively part of the function definition so for readability reasons there should be no blank lines here.

Also, don't just use `expected` for the argument names for the expected values for all the parameterised test functions. Use more descriptive names, like `expected_n_pix` or `expected_peak` or `image_size`. 

Note also that you can have as many arguments as you like, you don't have to use just two and start packing multiple values into them using tuples, and sometimes you might only need one if the only different between tests is which PSF you're testing.

**Comment by AnthonyHorton** [comment]
Also, try to get into the habit of running the tests on your own computer first so that you catch typos & other small errors before you commit & push.

**Comment by bazkiaei** [comment]
I am on it.
It is my bad that forgot to run the test.

**Comment by lspitler** [comment]
@AnthonyHorton just checking where this is at. Were you going to sit down with @bazkiaei and finalise this? Is there any way I can help?

**Comment by bazkiaei** [comment]
> @AnthonyHorton just checking where this is at. Were you going to sit down with @bazkiaei and finalise this? Is there any way I can help?

Maybe next Monday or Tuesday I go AAO to work on it?

**Comment by lspitler** [comment]
@AnthonyHorton is there a good day for @bazkiaei to visit the AAO this week?

**Pull request merged**